app/src/main/java/de/westnordost/streetcomplete/quests/police_type/AddPoliceType.kt

```diff
@@ -0,0 +1,32 @@
+package de.westnordost.streetcomplete.quests.police_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.quest.NoCountriesExcept
+
+class AddPoliceType : OsmFilterQuestType<PoliceType>() {
+
+    override val elementFilter = """
+        nodes, ways, relations with
+          amenity = police
+          and !operator
+    """
+    override val commitMessage = "Add police type"
+    override val wikiLink = "Tag:amenity=police"
+    override val icon = R.drawable.ic_quest_police
+    override val enabledInCountries = NoCountriesExcept(
+        "IT"
+    )
+
+    override fun getTitle(tags: Map<String, String>) = R.string.quest_policeType_title
+
+    override fun createForm() = AddPoliceTypeForm()
+
+    override fun applyAnswerTo(answer: PoliceType, changes: StringMapChangesBuilder) {
+        changes.add("name", answer.policeName);
```

Comment:
Are you sure about adding names? I am unfamiliar with Italy tagging, but in Poland police stations are often with own names such as `name=Komisariat IV`. See https://www.openstreetmap.org/way/201286959 https://www.openstreetmap.org/way/374532370 https://www.openstreetmap.org/way/167008548 that may be similar cases in Italy.

And additionally note that quest filter is not excluding things with `name` tag or `operator:wikidata` or `operator:wikipedia`

See say https://www.openstreetmap.org/way/24240779 - what will happen if user will solve quest?

Reply:
>Are you sure about adding names? I am unfamiliar with Italy tagging, but in Poland police stations are often with own names such as name=Komisariat IV. See https://www.openstreetmap.org/way/201286959 https://www.openstreetmap.org/way/374532370 https://www.openstreetmap.org/way/167008548 that may be similar cases in Italy.

I added the names only to avoid confusion between Polizia Stradale (road patrols), Polizia Ferroviaria (for trains) and Polizia di Stato (the main branch) which all have `operator=Polizia di Stato`. I agree, indeed I'd have never added them as me myself wouldn't add names with `amenity=police`. I've just asked on the Italian ML for this to see how other mappers add these. Anyway I think that the `name` tag doesn't get overridden with `add()` does it?
>And additionally note that quest filter is not excluding things with name tag or operator:wikidata or operator:wikipedia
Yes, that was one thing I was thinking about, I didn't filter it because I want to add `operator` anyway (which I think is more important), the only reason there are these two tags is if it is a very important police station operated exclusively by a separated entity only for that station. 
So I wouldn't like to override the tags but I'd still like to add the `operator` one, how do you think we can solve that? 🤔

Reply:
You asked on the Italian mailing list, so I'd say you should proceed whatever was the conclusion of that.
Note that it is also possible to only add the name if it does not exist yet:
`if (changes.getPreviousValue(...) == null) changes.add(...)`